src/MergeStrategy/Priority.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace SilverStripe\Config\MergeStrategy;
+
+use SilverStripe\Config\Collections\MutableConfigCollectionInterface;
+
+class Priority
+{
+    /**
+     * Merges an array of values into a collection
+     *
+     * @param array $mine Map of key to array with value and metadata sub-keys
+     * @param MutableConfigCollectionInterface $theirs
+     * @return MutableConfigCollectionInterface
+     */
+    public static function merge(array $mine, MutableConfigCollectionInterface $theirs)
```

Comment:
ahhhh I have a problem with adding static functions that I need to get over. I don't know why I didn't make it static to begin with